src/System.Management.Automation/engine/remoting/commands/PushRunspaceCommand.cs

```diff
@@ -172,6 +174,18 @@ public override PSCredential Credential
                    ParameterSetName = EnterPSSessionCommand.VMNameParameterSet)]
         public String ConfigurationName { get; set; }
 
+        #region Supress PSRemotingBaseCmdlet SSH hash parameter set
+
+        /// <summary>
+        /// Connection
```

Comment:
Yes.  Enter-PSSession cmdlet derives from a base cmdlet that defined SSHConnection parameter, which is not applicable here and must be suppressed.  This is done throughout our code base.